Mesh_3/include/CGAL/Mesh_edge_criteria_3.h

```diff
@@ -121,58 +122,38 @@ class Mesh_edge_criteria_3
 
   /// \name Creation
   /// @{
+
   /*!
   * returns an object to serve as criteria for edges.
   *
   * \param length_bound is an upper bound
   * for the length of the edges which are used to discretize the curves.
+  * It can be a functional or a constant.
```

Comment:
I am not completely sure about that wording. @janetournois?

Reply:
would you prefer "functor"?

Reply:
I do not know. What is the usual wording for the other Mesh_3 parameters that can take either a constant or a "field"?